src/libraries/System.ComponentModel.Annotations/src/System/ComponentModel/DataAnnotations/Schema/ColumnAttribute.cs

```diff
@@ -60,7 +60,7 @@ public int Order
         /// <summary>
         ///     The database provider specific data type of the column the property is mapped to.
         /// </summary>
-        public string TypeName
+        public string? TypeName
```

Comment:
I think this should stay nullable but have `[DisallowNull]` attribute

Reply:
Thanks for catching this, it was an oversight on my part. Will apply your suggestion.